x-pack/plugins/security_solution_serverless/common/pli/pli_config.ts

```diff
@@ -38,10 +38,10 @@ export const PLI_PRODUCT_FEATURES: PliProductFeatures = {
       ProductFeatureKey.endpointExceptions,
     ],
     complete: [
+      ProductFeatureKey.endpointHostIsolationExceptions,
       ProductFeatureKey.endpointResponseActions,
       ProductFeatureKey.osqueryAutomatedResponseActions,
       ProductFeatureKey.endpointAgentTamperProtection,
-      ProductFeatureKey.endpointExceptions,
```

Comment:
thanks 🙇

Reply:
@ashokaditya , is Endpoint Exceptions an Essentials feature?
cc: @caitlinbetz

Reply:
Yes endpoint exceptions is available to Endpoint Essentials/Complete